request/builder.js

```diff
@@ -635,118 +635,47 @@ class ContractExecutionRequestBuilder {
 }
 
 /**
- * Used for building a RequestProofRegistrationRequest
+ * Used for building a ExecutionValidationRequest
  */
-class RequestProofRegistrationRequestBuilder {
+class ExecutionValidationRequestBuilder {
   /**
    * @constructs
-   * @param {RequestProofRegistrationRequest} request
+   * @param {ExecutionValidationRequest} request
    */
   constructor(request) {
     this.request = request;
   }
 
   /**
-   * Sets the contract ID
-   * @param {string} id
-   * @return {RequestProofRegistrationRequestBuilder}
-   */
-  withContractId(id) {
-    /** @const */
-    this.contractId = id;
-    return this;
-  }
-
-  /**
-   * Sets the contract argument
-   * @param {string} argument
-   * @return {RequestProofRegistrationRequestBuilder}
-   */
-  withContractArgument(argument) {
-    /** @const */
-    this.contractArgument = argument;
-    return this;
-  }
-
-  /**
-   * Sets the ID of the certificate holder
-   * @param {string} id
-   * @return {RequestProofRegistrationRequestBuilder}
-   */
-  withCertHolderId(id) {
-    /** @const */
-    this.certHolderId = id;
-    return this;
-  }
-
-  /**
-   * Sets the certificate's version
-   * @param {number} version
-   * @return {RequestProofRegistrationRequestBuilder}
-   */
-  withCertVersion(version) {
-    /** @const */
-    this.certVersion = version;
-    return this;
-  }
-
-  /**
-   * Sets the signature
-   * @param {Uint8Array} signature
-   * @return {RequestProofRegistrationRequestBuilder}
+   * Sets the request
+   * @param {ContractExecutionRequest}
+   * @return {ExecutionValidationRequestBuilder}
    */
-  withSignature(signature) {
+   withContractExecutionRequest(request) {
```

Comment:
Sorry it's super minor. 
Does it have an additional leading space here?

Reply:
Oh, it's unnecessary. Thanks!